Toolset/palettes/standalone settings/revstandalonesettingsiosbehavior.livecodescript

```diff
@@ -25,7 +25,7 @@ on initialize
       updateSettings
    else
       set the enabled of button "Build for iOS" to false
-      set the enabled of group "iOS Settings" to false
+      -- set the enabled of group "iOS Settings" to false
    end if
```

Comment:
I noticed that now there is no group "iOS Settings". The problem is that if we are not on Mac, we have to disable everything on the iOS card (since you cannot build for iOS if you are not on Mac). So probably you have to create a group "iOS Settings" and put *everything* in that group, except from the button "Build for iOS". And of course, uncomment this line (`set the enabled of group "iOS Settings" to false`) once you do that, and re-examine if any removed reference to group "iOS Settings" is still needed.